src/clr-angular/button/button-group/button.spec.ts

```diff
@@ -151,22 +151,21 @@ export default function(): void {
             });
 
             it("implements LoadingListener", () => {
-                expect(componentInstance.button1.startLoading).toBeDefined();
-                expect(componentInstance.button1.doneLoading).toBeDefined();
+                expect(componentInstance.button1.loadingStateChange).toBeDefined();
             });
 
             it("sets loading to true on start loading", () => {
                 expect(componentInstance.button1.loading).toBeUndefined();
 
-                componentInstance.button1.startLoading();
+                componentInstance.button1.loadingStateChange(ClrLoadingState.LOADING);
 
                 expect(componentInstance.button1.loading).toBe(true);
             });
 
             it("sets loading to false on done loading", () => {
                 expect(componentInstance.button1.loading).toBeUndefined();
 
-                componentInstance.button1.doneLoading();
+                componentInstance.button1.loadingStateChange(ClrLoadingState.DEFAULT);
```

Comment:
Are we missing a test for `ClrLoadingState.SUCCESS`?

Reply:
More extensive testing is done in `loading-button.spec.ts` including test for `ClrLoadingState.SUCCESS` :)